Paper.tex

```diff
@@ -1418,6 +1418,13 @@ \section{Signing Transactions}\label{app:signing}
 %\mathtt{\tiny secp256k1p} &= 2^{256} - 2^{32} - 977\\
 \end{align}
 
+Note that $\mathtt{\small ECDSARECOVER}$ function does not follow the distinction of valid or invalid transactions defined here. $\mathtt{\small ECDSARECOVER}$ returns non-empty results if the following conditions are met (this is relevant in the definition of $\Xi_{\mathtt{ECREC}}$).
+\begin{align}
+0 < r &< \mathtt{\tiny secp256k1n} \\
+0 < s &< \mathtt{\tiny secp256k1n} \\
+v &\in \{27,28\}
```

Comment:
This conflicts with EIP-155 and https://ethereum.github.io/yellowpaper/paper.pdf#v.

Reply:
You may also notice "These are formally defined in theliterature, e.g. by ?." #630 should fix that.

Reply:
Would be good to add to the build script to pick up Latex errors.

Reply:
I think I understand the conflict. Following are the two definitions of an invalid signature.

![screenshot from 2018-02-21 19-28-07](https://user-images.githubusercontent.com/16969914/36470011-6e5f5080-173d-11e8-98e8-7640bcb19552.png)

![screenshot from 2018-02-21 19-29-26](https://user-images.githubusercontent.com/16969914/36470040-9105d01e-173d-11e8-8870-ebdeec146e9b.png)

Reply:
Wouldn't it also be good to update the implementations and the first of the above?

Reply:
Noticed something else: #632.

Reply:
Wouldn't it be better to update the EIP draft, implementations and then this paper for [the first screenshot with the ECREC precompile](https://ethereum.github.io/yellowpaper/paper.pdf#page.20) so that an invalid signature is defined in accordance with the second screenshot in [Appendix F](https://ethereum.github.io/yellowpaper/paper.pdf#appendix.F)?

Reply:
That will be a hardfork, so, there needs to be a separate EIP proposing the fix.

Reply:
#893